pkg/vpcmodel/nodesConnectivity.go

```diff
@@ -78,6 +78,29 @@ func (c *VPCConfig) getLoadBalancerRule(src, dst Node) LoadBalancerRule {
 	return nil
 }
 
+func (c *VPCConfig) getPrivateSubnetRule(src, dst Node) PrivateSubnetRule {
+	switch {
+	case dst.IsInternal():
+		return dst.(InternalNodeIntf).Subnet().GetPrivateSubnetRule(src, dst)
+	case src.IsInternal():
+		return src.(InternalNodeIntf).Subnet().GetPrivateSubnetRule(src, dst)
+	}
+	return nil
+}
+
+// getOtherRulesConn() return the connectivity of all rules that are not part of the filters and routers.
+func (c *VPCConfig) getOtherRulesConn(src, dst Node, isIngress bool) *connection.Set {
```

Comment:
I don't like the term `other` here.
Perhaps` NonFilterNonRouter`?

Reply:
fixed